shell/browser/api/electron_api_menu_mac.mm

```diff
@@ -155,8 +166,26 @@
   // be done manually.
   base::mac::ScopedSendingEvent sendingEventScoper;
```

Comment:
`ui::showContextMenu` seems to create `ScopedSendingEvent` and `ScopedAllowApplicationTasksInNativeNestedLoop` if `allow_nested_tasks = true`, can these be removed in favor of that ?

Reply:
I opted for explicitly doing it on our side of the contract instead of relying on a boolean flag in Chromes internals 🤔 Not fussed either way, just wanted as small a lift as possible.